apps/communications/manifest.webapp

```diff
@@ -113,13 +113,21 @@
     },
     "open": {
       "filters": {
-        "type": ["webcontacts/contact", "text/vcard", "text/x-vcard",
+        "type": ["text/vcard", "text/x-vcard",
                  "text/directory"]
       },
       "disposition": "inline",
       "href": "/contacts/index.html?open",
       "returnValue": true
     },
+    "open": {
```

Comment:
This is not working as expected. In [1] they explain that now this must be an array as `open: [ {//filter 1}, ....,  {//filter n}]`. In order to ensure that this is working as expected, change the order (first use the `webcontacts/contact` and the `vcard` filter).

[1] https://bugzilla.mozilla.org/show_bug.cgi?id=1176712#c2